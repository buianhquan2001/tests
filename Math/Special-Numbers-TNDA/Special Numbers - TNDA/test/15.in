100
918610467 224652929
539249385 836852479
605937755 425846115
250609253 118109681
844842259 919633714
213639788 846550506
303881635 493625972
994894850 667053824
173762573 785980114
16534889 288217527
186221788 148953666
475267227 77174494
698250755 733488808
406673229 285649016
465573207 938484705
881200554 490039196
768584047 763393713
928361076 12215758
524020160 237526925
931789543 847300267
288303930 27929591
158062030 114908926
568135232 738051896
420697503 9950812
162241499 106611326
405821994 4643048
112478875 613578906
143252172 827336803
985367627 81216500
438171597 215923326
311808790 334446402
196609837 60789854
793405583 338411886
146994184 659665610
937256835 704145084
670085173 134223145
697004038 449360015
276125847 133328036
532943558 98705496
776740615 287043170
874137638 345675631
882747627 930466908
159538099 706260146
126293814 518636052
168753594 58100928
892487644 919593475
670422197 720167497
737171221 505248531
57101988 750581840
964335323 372871420
430004601 529848310
322945439 839103239
123056439 838807466
805836993 510111977
869788100 571493855
379202170 869963885
988226344 345769906
763442790 975479289
768395968 765984958
428938546 533565605
554879345 817222124
568982704 317143216
902105025 452495927
120066912 217498380
593562025 328555003
282769146 681490837
779497614 426551978
70944959 981934175
520690687 187185906
163181566 748176505
770735673 6466042
833434279 137450073
973436074 662209718
825942166 290978955
205802842 511266957
102744284 346825635
668271177 153455715
492371728 818427655
751112532 959468990
997225252 483939701
769352412 837234744
374717498 772376341
432128894 357742796
416901269 989484203
988966845 430820009
7623496 495590583
660890683 758934405
172450923 69385141
242167815 784832103
849324493 170200212
782988489 599689109
202169290 136174735
643187406 994180244
223780798 999318542
638012336 439962218
889715891 239204540
609229137 781825452
553494532 208016268
266535929 538417784
484585070 787859935
