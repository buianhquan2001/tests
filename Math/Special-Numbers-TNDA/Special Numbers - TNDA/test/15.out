318387580
820075810
41334489
255169536
996801399
42476355
633225680
554825229
742986295
572884544
716454485
665226258
266421289
850091128
31633389
9847684
125056350
413608729
687583346
592738796
67916216
330689313
134915039
433795354
481106075
882387939
88709374
202829566
663574019
222398536
11825721
632277958
581318818
934876102
549128381
897547606
612609953
868885900
899863770
412458508
674360173
290050007
221724463
429096066
676184483
342498738
562876310
405916390
956060679
966600601
919871606
591701090
591936736
579628865
194452014
125389039
373549510
485880028
557414978
951459602
100013740
840325176
641605859
184259802
204677979
292617516
721279991
85941116
725237105
24939032
714156595
867570951
442659431
373448683
627642330
17026968
834683875
704329723
16575450
890073580
495897247
824451662
635462785
112858013
78882721
635230936
684744368
381852040
695922387
300649773
33189580
238603319
625214971
855510624
823517928
907877977
552855136
17623343
244498843
94419127
